canonical-web-and-design/maas-ui
# Fix VM compose storage capacity input.

## Done
- I broke the storage charts when updating the compose form validation because I accidentally changed the input type to text, which screwed with some calculations. This reverts it back to number, and imposes a minimum of 0.
- Increased the storage cell width so the text doesn't cut off.

## QA
- Open KVM compose form, change the storage values and see that the bar charts match the values.
